test/e2e.method.signing.js

```diff
@@ -9,6 +9,14 @@ describe('transaction and message signing [ @E2E ]', function() {
     let web3;
     let accounts;
     let wallet;
+    let basic;
+    let instance;
+
+    var basicOptions = {
```

Comment:
could you use let or const?

Reply:
Fixed in 269d11943c0db30f11effb227856ecb473bec07c